ivy_tests/test_ivy/test_frontends/test_jax/test_jax_src_api.py

```diff
@@ -4,6 +4,83 @@
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_frontend_test
 from ivy.functional.frontends.jax._src.api import device_put, device_get
+from hypothesis import given, assume, strategies as st
+from ivy.functional.frontends.jax._src.api import vmap
+
+
+def _fn1(x, y):
+    return ivy.matmul(x, y)
+
+
+def _fn2(x, y):
+    return ivy.vecdot(x, y)
+
+
+def _fn3(x, y):
+    return ivy.add(x, y)
+
+
+# vmap
+@handle_frontend_test(
+    fn_tree="jax._src.api.vmap",
+    func=st.sampled_from([_fn1, _fn2, _fn3]),
+    dtype_and_arrays_and_axes=helpers.arrays_and_axes(
+        allow_none=False,
+        min_num_dims=2,
+        max_num_dims=5,
+        min_dim_size=2,
+        max_dim_size=10,
+        num=2,
+        return_dtype=True,
+    ),
+    in_axes_as_cont=st.booleans(),
+)
+def test_vmap(func, dtype_and_arrays_and_axes, in_axes_as_cont):
+    dtype, generated_arrays, in_axes = dtype_and_arrays_and_axes
+    arrays = [ivy.native_array(array) for array in generated_arrays]
+
+    if in_axes_as_cont:
+        vmapped_func = vmap(func, in_axes=in_axes, out_axes=0)
+    else:
+        vmapped_func = vmap(func, in_axes=0, out_axes=0)
+
+    assert callable(vmapped_func)
+
+    try:
+        fw_res = helpers.flatten_and_to_np(ret=vmapped_func(*arrays))
+        fw_res = fw_res if len(fw_res) else None
+    except Exception:
+        fw_res = None
+
+    ivy.set_backend("jax")
+    arrays = [ivy.native_array(array) for array in generated_arrays]
+    if in_axes_as_cont:
+        jax_vmapped_func = ivy.vmap(func, in_axes=in_axes, out_axes=0)
+    else:
+        jax_vmapped_func = ivy.vmap(func, in_axes=0, out_axes=0)
+
+    assert callable(jax_vmapped_func)
```

Comment:
Frontend functions should be compared with their corresponding native functions, not ivy functions. So in this case you need to compare `ivy.functional.frontends.jax._src.api.vmap` to the original `jax._src.api. vmap`.

Reply:
Got it